File: ChatApp.Server/Services/ViewModels/RequestDetailsModel.cs
namespace ChatApp.Server.Services.ViewModels
{
    public class RequestDetailsModel
    {
        public int Id { get; set; }

        public AddUserModel UserFrom { get; set; }

        public AddUserModel UserTo { get; set; }

        public int RequestStatus { get; set; }

        public bool IsDeleted { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime ModifiedAt { get; set; }
    }
}

File: chatapp.client/src/app/models/request-details-model.ts
import { AddContactModel } from "./add-contact-model";

export class RequestDetailsModel {
    id: number = 0;
    userFrom: AddContactModel;
    userTo: AddContactModel;
    requestStatus: number = 0;
    isDeleted: boolean = false;
    createdAt: string = '';
    modifiedAt: string = '';
}